# a=10
# b=0
# # try:
# #     c=a/b
# # except (ZeroDivisionError,TypeError) as e:
# #     print(f"Error :{e}")
# # else:
# #     print("No error occured")
# # finally:
# #     print("This runs even when an error has been occured")


# a=10
# b=2
# if(b==0):
#     raise ZeroDivisionError("Denominator cannot be zero")
# print(a//b)


# class Myerror(Exception):
#     pass

# voter=19
# if(voter<=18):
#     raise Myerror("Ineligible")
# print("Eligible")

a=10
b=2
assert b<0,"Ok"
print("Hi")


